### ISSUE TITLE
minimesos state does not pretty print

### ISSUE THREAD

**Issue by frankscholten** [ContainerSolutions/minimesos#231]

**Comment by sadovnikov** [comment]
@frankscholten What do you exactly mean?
```
vikmac:mini-mesos vik$ ./bin/minimesos state
{
  "git_sha": "2dd7f7ee115fe00b8e098b0a10762a4fa8f4600f",
  "leader": "master@172.17.0.4:5050",
  "slaves": [{
    "used_resources": {
      "disk": 0,
      "mem": 0,
      "cpus": 0
    },
    "hostname": "172.17.0.6",
    "offered_resources": {
      "disk": 0,
....
}
```

**Comment by frankscholten** [comment]
Hmm perhaps I was using an older version. I got a compact JSON string

**Comment by sadovnikov** [comment]
resolved by #233
